protocol/messenger_communities.go

```diff
@@ -532,6 +535,87 @@ func (m *Messenger) updateCommunitiesActiveMembersPeriodically() {
 	}()
 }
 
+func (m *Messenger) HandleCommunityGrantUpdated(state *ReceivedMessageState, update *protobuf.CommunityGrantUpdated, statusMessage *v1protocol.StatusMessage) error {
+	community, err := m.communitiesManager.GetByID(update.CommunityId)
+	if err != nil {
+		return err
+	}
+
+	if community == nil {
+		return communities.ErrOrgNotFound
+	}
+
+	return m.communitiesManager.HandleCommunityGrant(community, update.Grant, update.Clock)
+}
```

Comment:
Actually, maybe we should just check if community exists right in `communityManager.HandleCommunityGrant`?  🤔

Reply:
Not sure, `HandleCommunityGrant` used [here](https://github.com/status-im/status-go/blob/550d74405795e9f419a426c6310929027f5bd48b/protocol/communities/manager.go#L2983) and it's always present in this case

Reply:
Would make sense to me anyway 🙂
But up to you, not a big deal really.